lib/logstash/inputs/logstash.rb

```diff
@@ -10,7 +10,7 @@
 class LogStash::Inputs::Logstash < LogStash::Inputs::Base
   include LogStash::PluginMixins::PluginFactorySupport
 
-  config_name "logstash"
+  config_name "logstash_input"
```

Comment:
🚫 This defies the convention of plugins having a `config_name` that matches the name of the plugin.